static.config.js

```diff
@@ -21,6 +22,20 @@ const analyticsCode = `
 })(window,document,'https://static.hotjar.com/c/hotjar-','.js?sv=');
 `
 
+const generateCombinationsFromArray = (array, property, separator = '-') => {
+  let results = []
+  for (let i = 0; i <= array.length - 1; i++) {
+    for (let j = 0; j <= array.length - 1; j++) {
+      if (array[i][property] !== array[j][property]) {
+        results.push(`${array[i][property]}${separator}${array[j][property]}`)
+      }
+    }
+    if (i == array.length - 1) {
+      return results
+    }
+  }
+}
+
```

Comment:
You'll need to update the Document title and description to differentiate the pages in search results. See the [routeInfo](https://github.com/nozzle/react-static/blob/master/docs/config.md#document) param passed to Document for access to the current pair